### PR TITLE
CleanUp  & TC added for L2_pool Graph transformation.

### PR DESCRIPTION
**Pull request by amitsrivastava78**
Clean-up: replace FindOperator with FindOp and add missing unit tests
for the L2_pool graph transformation.

### CODE REVIEW

File: tensorflow/lite/toco/graph_transformations/identify_l2_pool.cc [hunk 1]
```diff
@@ -105,9 +91,9 @@ ::tensorflow::Status IdentifyL2Pool::Run(Model* model, std::size_t op_index,
   model->EraseArray(sqrt_op->inputs[0]);
 
   // Erase three operators being replaced.
-  model->operators.erase(FindOperator(model, square_op));
-  model->operators.erase(FindOperator(model, avpool_op));
-  model->operators.erase(FindOperator(model, sqrt_op));
+  model->operators.erase(FindOp((*model), square_op));
```
Comment (multiverse-tf) [inline tensorflow/lite/toco/graph_transformations/identify_l2_pool.cc]: Similar to my comment in #27616, the "()" enclosing "*model" looks redundant to me.

**Review by multiverse-tf — COMMENTED** [review]
What about changing the description of the commit to be more specific? Like: "Clean-up: replace FindOperator with FindOp and add missing unit tests for the L2_pool graph transformation."? Note that I think the change is essentially a clean-up not a bug fix.

File: tensorflow/lite/toco/graph_transformations/identify_l2_pool.cc (hunk 1 above)
Comment (amitsrivastava78) [reply tensorflow/lite/toco/graph_transformations/identify_l2_pool.cc]: @multiverse-tf , thanks for the review i have updated as per your suggestion, kindly check.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@multiverse-tf , thanks for the review i have updated as per your suggestion, kindly check.

Regards
Amit

**Review by multiverse-tf — APPROVED** [review]

**Comment by amitsrivastava78** [comment]
@multiverse-tf , thanks for approving the PR.
@gbaned , @rthadur please change the PR label to approved.

Regards
Amit

**Pull request merged**